airflow/cli/commands/worker_command.py

```diff
@@ -59,6 +62,12 @@ def worker(args):
     if conf.has_option("celery", "pool"):
         options["pool"] = conf.get("celery", "pool")
 
+    if skip_serve_logs is False:
+        print(
+            "Starting serve logs process within worker is going to be deprecated, "
```

Comment:
Are we sure we want to deprecate this in 2.0? One of the current principles of Airflow is to make it work out of the box so this log serving may still be a good default to have.

If we want to keep the warning should probably use log. syntax instead of print()

Reply:
In that case, we can remove that warning.

Reply:
removed

Reply:
Looks like the print statement still shows the deprecation comment.

Reply:
@aoen I've removed the whole warning itself.